pootle/static/js/editor/app.js

```diff
@@ -2315,4 +2373,52 @@ PTL.editor = {
     return true;
   },
 
+  toggleSuggestion(e, { canHide = false } = {}) {
+    e.stopPropagation();
+    const suggestionId = parseInt(e.currentTarget.dataset.suggId, 10);
+    const suggestion = document.getElementById(`suggestion-${suggestionId}`);
+
+    if (this.selectedSuggestionId === undefined) {
+      this.selectedSuggestionId = suggestionId;
+      const props = {
+        suggId: this.selectedSuggestionId,
+        initialSuggestionText: e.currentTarget.dataset.translationAid,
+        localeDir: this.settings.localeDir,
+        onAcceptSuggestion: this.handleAcceptSuggestion,
```

Comment:
To have a consistent behavior across component event handlers, it'd be preferable to bind these handlers via `this.handleMethod.bind(this)`, and this way it'll also be possible to refer to `this` in the handlers instead of `PTL.editor`.